src/main/scala/com/cognite/spark/datasource/CdpRelation.scala

```diff
@@ -17,10 +18,15 @@ object Setter {
 }
 case class NonNullableSetter[A](set: A)
 
+case class DeleteItem(
+    id: Long
+)
+
 abstract class CdpRelation[T: Decoder](config: RelationConfig, shortName: String)
     extends BaseRelation
     with TableScan
-    with Serializable {
+    with Serializable
+    with CdpConnector {
```

Comment:
Now that this is added in the CdpRelation we can remove this from all subclasses. Please include FilesRelation and RawTableRelation as well, and any others.

Reply:
Will fix for FilesRelation. Looks like RawTableRelation does not extend CdpRelation, but BaseRelation directly.